File: seed/csharp-sdk/exhaustive/explicit-namespaces/src/SeedApi/Reqwithheaders/Requests/ReqWithHeadersGetWithCustomHeaderRequest.cs
using global::System.Text.Json.Serialization;
using SeedApi.Core;

namespace SeedApi.Reqwithheaders;

[Serializable]
public record ReqWithHeadersGetWithCustomHeaderRequest
{
    [JsonIgnore]
    public required string TestEndpointHeader { get; set; }

    [JsonIgnore]
    public required string Body { get; set; }

    /// <inheritdoc />
    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: seed/php-sdk/exhaustive/no-custom-config/src/Reqwithheaders/Requests/ReqWithHeadersGetWithCustomHeaderRequest.php
<?php

namespace Seed\Reqwithheaders\Requests;

use Seed\Core\Json\JsonSerializableType;

class ReqWithHeadersGetWithCustomHeaderRequest extends JsonSerializableType
{
    /**
     * @var string $testEndpointHeader
     */
    public string $testEndpointHeader;

    /**
     * @var string $body
     */
    public string $body;

    /**
     * @param array{
     *   testEndpointHeader: string,
     *   body: string,
     * } $values
     */
    public function __construct(
        array $values,
    ) {
        $this->testEndpointHeader = $values['testEndpointHeader'];
        $this->body = $values['body'];
    }
}
